spec/requests/institutions_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'swagger_helper'
+
+describe 'Institutions API' do
+    path "/institutions" do

```

Comment:
It seems like this file uses four space long indents, while all the other files use two spaces